spec/models/miq_product_feature_spec.rb

```diff
@@ -4,7 +4,7 @@
 
 describe MiqProductFeature do
   before do
-    @expected_feature_count = 872
+    @expected_feature_count = 878
```

Comment:
@martinpovolny @Ladas I don't like that we have to do this whenever we add features.  Is there any value to you for the single assertion below that uses this counter?